<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>684</width>
    <height>508</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Analizador Sintáctico</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>8</y>
      <width>431</width>
      <height>81</height>
     </rect>
    </property>
    <property name="title">
     <string>Entrada</string>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Expresión</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEdit_expresion"/>
     </item>
     <item row="1" column="0" colspan="2">
      <widget class="QPushButton" name="pushButton_analizar">
       <property name="text">
        <string>Analizar</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>431</width>
      <height>361</height>
     </rect>
    </property>
    <property name="title">
     <string>Representación Intermedia (Arreglo)</string>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QTableWidget" name="tableWidget_representacionIntermedia">
       <attribute name="horizontalHeaderVisible">
        <bool>true</bool>
       </attribute>
       <attribute name="horizontalHeaderCascadingSectionResizes">
        <bool>false</bool>
       </attribute>
       <attribute name="horizontalHeaderHighlightSections">
        <bool>true</bool>
       </attribute>
       <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
        <bool>false</bool>
       </attribute>
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <column>
        <property name="text">
         <string>Posición</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Etiqueta</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Argumento 1</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Argumento 2</string>
        </property>
       </column>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>10</y>
      <width>231</width>
      <height>451</height>
     </rect>
    </property>
    <property name="title">
     <string>Tabla de Símbolos</string>
    </property>
    <widget class="QTableWidget" name="tableWidget_simbolos">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>21</y>
       <width>211</width>
       <height>421</height>
      </rect>
     </property>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <column>
      <property name="text">
       <string>Posición</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Identificador</string>
      </property>
     </column>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>684</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
